zcash_client_sqlite/src/wallet.rs

```diff
@@ -712,15 +713,21 @@ pub(crate) fn get_sapling_commitment_tree<P>(
 pub(crate) fn get_sapling_witnesses<P>(
     wdb: &WalletDb<P>,
     block_height: BlockHeight,
-) -> Result<Vec<(NoteId, IncrementalWitness<Node>)>, SqliteClientError> {
+) -> Result<
+    Vec<(
+        NoteId,
+        IncrementalWitness<Node, SAPLING_COMMITMENT_TREE_DEPTH>,
+    )>,
+    SqliteClientError,
+> {
```

Comment:
```suggestion
) -> Result<Vec<(NoteId, sapling::IncrementalWitness)>, SqliteClientError> {
```

Reply:
Point of order for future reviews: rather than taking the time to add a suggestion for each one, it's much better to just leave a single blocking comment; that way I can integrate the changes and address any build/formatting issues at the same time.